domain/src/main/java/com/exalt/domain/home/usecases/GetOwnerUseCase.kt

```diff
@@ -1,13 +1,13 @@
 package com.exalt.domain.home.usecases
 
 import com.exalt.domain.home.models.OwnerModel
-import com.exalt.domain.home.repositories.UserRepository
+import com.exalt.domain.home.repositories.OwnerRepository
 import javax.inject.Inject
 
 class GetOwnerUseCase @Inject constructor(
-    private val userRepository: UserRepository
+    private val ownerRepository: OwnerRepository
 ) {
     suspend fun invoke(id: String): OwnerModel? = runCatching {
-        userRepository.getUserBy(id)
+        ownerRepository.getUserBy(id)
```

Comment:
Suite à mon commentaire plus haut, le code devrait être du style :
```suggestion
        val owner = ownerRepository.getUserBy(id)
        val ownerAge = getOwnerAge(owner.dateOfBirth)
        if (ownerAge < LEGAL_AGE) {
            return owner.copy(pictureUrl = "")
        }
        return owner
```